extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -97,10 +97,25 @@
             "outputs": [],
             "execution_count": null
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## 3. Format toc.yml file of the book\r\n",
+                "\r\n",
+                "Create a valid toc.yml file that enables your book to be opened on our view-let.\r\n",
+                "\r\n",
+                "<span style=\"color:red\">Note:</span> Skipping this step would have ill-formed toc and need your manual updates to it."
```

Comment:
Might want to reword this for greater clarity. People using this notebook probably wouldn't know anything about the Book file structure.